graphql/http_handler.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/99designs/gqlgen/graphql"
 	"github.com/99designs/gqlgen/graphql/handler"
 	"github.com/99designs/gqlgen/graphql/handler/apollotracing"
+	generated "github.com/evergreen-ci/evergreen/graphql/generated"
```

Comment:
I don't think we need a generated package it is only used by graphql why do we need a separate package to store these values?

Reply:
if the `/resolvers` folder is kept (which I'm not sure it is based on your previous comment), then moving this out will cause an import cycle (because the files outside of `/resolvers` import `/resolvers` but then `/resolvers` needs to import the files outside of itself)